src/cmd/services/m3coordinator/downsample/id_pool_types.go

```diff
@@ -35,8 +36,8 @@ import (
 
 var (
 	defaultMetricNameTagName = []byte(model.MetricNameLabel)
-	rollupTagName            = []byte("__rollup__")
-	rollupTagValue           = []byte("true")
+	rollupTagName            = []byte(m3model.RollupTagName)
```

Comment:
lol now I find `m3model` even more confusing, since we have `m3_rollup` in `src/metrics`

maybe import as `coordmodel` ?